### PR TITLE
Add misusage description of token

### PR DESCRIPTION
**Pull request by mcd500**
Adding description of token expiring at the issue https://github.com/ietf-teep/teep-protocol/issues/83.

@dthaler please review

### CODE REVIEW

File: draft-ietf-teep-protocol.md [hunk 1]
```diff
@@ -335,6 +335,11 @@ token
   generated by a TAM MUST be randomly created.
   Consequent token values MUST be different for each request message
   to distinguish the correct response from multiple requests.
+  The token value MUST NOT be used for other purposes, such as a TAM to
+  identify the devices and/or a device to identify TAMs or Trusted Components.
+  The TAM MUST expire the token value after receiving the first responce
+  from the device and ignore any consequent messages which have the same token
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:341]:
```suggestion
  from the device and ignore any subsequent messages that have the same token
```

File: draft-ietf-teep-protocol.md [hunk 2]
```diff
@@ -335,6 +335,11 @@ token
   generated by a TAM MUST be randomly created.
   Consequent token values MUST be different for each request message
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:336]:
```suggestion
  Subsequent token values MUST be different for each request message
```

**Review by dthaler — APPROVED** [review]
Ok to merge after accepting my suggestions

**Comment by mcd500** [comment]
@dthaler thanks, merging

**Pull request merged**